.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "datamodel"]
-	path = datamodel
-	url = git@github.com:QGEP/datamodel.git
+[submodule "plugin_ci"]
+	path = plugin_ci
+	url = https://github.com/opengisch/plugin_ci.git
```

Comment:
why?

Reply:
can you use new library instead?

Reply:
It's not a plugin, it's just individual scripts that are used. I'm not sure how complicated the upgrade would be or if it would be easier to move over those scripts.

Reply:
If it's just to upload the release asset, you'd better go with these 3 lines:

```python
repo = Github(github_token).get_repo(slug)
gh_release = repo.get_release(id=release_tag)
gh_release.upload_asset(path=asset_path, label=asset_name, name=asset_name)
```

using PyGithub.

Reply:
Like in the latest commit?